net-http-persistent.gemspec

```diff
@@ -0,0 +1,11 @@
+Gem::Specification.new 'net-http-persistent', '3.0.1' do |s|
```

Comment:
Is there a way not hard code this version?  I think a grep + sed might do it.

Reply:
added bump, so now `rake bump:minor` will update the version (and make a version bump commit)